docs/content/Schema/Getting-Started-with-YAML.mdx

```diff
@@ -0,0 +1,207 @@
+---
+title: Getting Started with YAML
+permalink: /schema/getting-started/yaml
+category: Reference
+menuOrder: 2
+---
+
+<WarningBox heading={`🐣  Preview`}>
+
+YAML modelling is currently in Preview, and the syntax may change in a future
+version.
+
+</WarningBox>
+
+## Ensure deployment is using latest Cube
+
+First, ensure you're using Cube version `0.31.0` or higher. Then create a new
+`.yml` file in the `schema/` folder.
+
+## Syntax
+
+<InfoBox>
+
+1. All schema properties can be written in either camel-case or snake-case i.e.
```

Comment:
Let's say while you can use both snake case and camel case, we recommend using snake case with YAML.